images/build/Dockerfiles/gitHubActions.Dockerfile

```diff
@@ -118,6 +118,17 @@ COPY --from=nodetools-install /opt /opt
 COPY --from=buildscriptgenerator /opt/buildscriptgen/ /opt/buildscriptgen/
 RUN ln -s /opt/buildscriptgen/GenerateBuildScript /opt/oryx/oryx
 
+# Install PHP pre-reqs
```

Comment:
Adding these did not cause any noticeable size increase to the image...we care about this because this image is going to be used in GitHub actions

Reply:
should we also install the prereqs in lts-versions image? we dont have any php version currently in lts-version build image